packages/alert-banner/index.tsx

```diff
@@ -43,41 +39,23 @@ function AlertBanner({
   text,
   url,
 }: AlertBannerProps): React.ReactElement {
-  const dismissalCookieId = `banner_${name || slugify(text, { lower: true })}`
-
   const [isShown, setIsShown] = useState(() => {
     const hasExpired = expirationDate && Date.now() > Date.parse(expirationDate)
-    const hasBeenDismissed =
-      typeof window === 'undefined'
-        ? false
-        : cookie.get(dismissalCookieId) === '1'
 
-    return !hasExpired && !hasBeenDismissed
+    return !hasExpired
   })
   const { themeClass } = useProductMeta(product)
 
   /**
-   * On mount, hide the banner if it is expired or
-   * if a cookie indicates it was previously closed
+   * On mount, hide the banner if it is expired
    */
   useEffect(() => {
-    const hasBeenDismissed = cookie.get(dismissalCookieId)
     const hasExpired = expirationDate && Date.now() > Date.parse(expirationDate)
-    const shouldBeShown = !hasBeenDismissed && !hasExpired
+    const shouldBeShown = !hasExpired
```

Comment:
Another non-blocking nit: I could be wrong but, looking at the logic now, I _think_ we could remove this const and just use `hasExpired` in `setIsShown`. Again, totally non-blocking!